web/source/osk/oskManager.ts

```diff
@@ -0,0 +1,1530 @@
+/// <reference path="../kmwexthtml.ts" />  // Includes KMW-added property declaration extensions for HTML elements.
+/// <reference path="../kmwstring.ts" />  // Includes KMW string extension declarations.
+// Includes the default layout specification.
+/// <reference path="defaultLayouts.ts" /> 
+// Includes the touch-mode language picker UI.
+/// <reference path="languageMenu.ts" />
+// Generates the visual keyboard specific to each keyboard.  (class="kmw-osk-inner-frame")
+/// <reference path="visualKeyboard.ts" />
+
+/***
+   KeymanWeb 10.0
```

Comment:
Let's make sure new files have correct initial version + copyright date

Reply:
I was planning to wait until after the next merge from Beta -> Alpha, so I can pre-emptively set the files to Keyman 12.0.  That sound fine?

Reply:
OK